### PR TITLE
Update README to point to RAML 1.0

### PR DESCRIPTION
**Pull request by dmartinezg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,9 @@
 [![Build Status](https://travis-ci.org/raml-org/raml-js-parser.png)](https://travis-ci.org/raml-org/raml-js-parser)
 [![Dependency Status](https://david-dm.org/raml-org/raml-js-parser.png)](https://david-dm.org/raml-org/raml-js-parser)
 
-This project contains a RAML parser capable (at this moment) of parsing version 0.8
-of the [RAML specification](http://raml.org/spec.html). The parser is written in CoffeeScript and its capable
-of running inside NodeJS as well as in-browser.
+This is the JS parser for RAML 0.8 as defined in the [0.8 RAML specification](https://github.com/raml-org/raml-spec/blob/master/raml-0.8.md)
```
Comment (usarid) [inline README.md]: This is a JavaScript parser for [RAML](http://raml.org) version 0.8 as defined in the [0.8 RAML specification](https://github.com/raml-org/raml-spec/blob/master/raml-0.8.md).

File: README.md [hunk 2]
```diff
@@ -4,23 +4,9 @@
...
-```
-x.y.z
-```
-
-in which *x.y* denotes the version of the [RAML specification](http://raml.org/spec.html)
-and *z* is the version of the parser.
-
-So *0.1.2* is the 2nd revision of the parser for the *0.1* version
-of the [RAML specification](http://raml.org/spec.html).
+If you want to try out the new features of RAML 1.0, you can try the new [JS parser](https://github.com/raml-org/raml-js-parser-2)
```
Comment (usarid) [inline README.md]: A newer version is now available [as a beta](https://github.com/raml-org/raml-js-parser-2). It supports RAML 1.0 as well as RAML 0.8.

**Comment by dmartinezg** [comment]
@usarid, can you help me proof read the changes with your feedback in?

File: README.md [hunk 3]
```diff
@@ -4,23 +4,9 @@
...
-
-in which *x.y* denotes the version of the [RAML specification](http://raml.org/spec.html)
-and *z* is the version of the parser.
-
-So *0.1.2* is the 2nd revision of the parser for the *0.1* version
-of the [RAML specification](http://raml.org/spec.html).
+A newer [version](https://github.com/raml-org/raml-js-parser-2) is now available as a beta. It supports RAML 1.0 as well as RAML 0.8.
 
 ### Contributing
 If you are interested in contributing some code to this project, thanks! Please submit a [Contributors Agreement](https://api-notebook.anypoint.mulesoft.com/notebooks#bc1cf75a0284268407e4) acknowledging that you are transferring ownership.
```
Comment (usarid) [inline README.md]: If you are interested in contributing some code to this project, thanks! Please first [read and accept the Contributors Agreement](https://api-notebook.anypoint.mulesoft.com/notebooks#bc1cf75a0284268407e4).

**Comment by usarid** [comment]
OK I made one more small change, now it looks good to go.

**Pull request merged**